pytest_selenium/safety.py

```diff
@@ -63,8 +63,9 @@ def _skip_sensitive(request, sensitive_url):
     """Skip destructive tests if the environment is considered sensitive"""
     destructive = 'nondestructive' not in request.node.keywords
     if sensitive_url and destructive:
-        pytest.skip(
-            'This test is destructive and the target URL is '
-            'considered a sensitive environment. If this test is '
-            'not destructive, add the \'nondestructive\' marker to '
-            'it. Sensitive URL: {0}'.format(sensitive_url))
+        message = 'This test is destructive and the target URL is '\
+            'considered a sensitive environment. If this test is '\
+            'not destructive, add the \'nondestructive\' marker to '\
+            'it. Sensitive URL: {0}'.format(sensitive_url)
+        print(message)
```

Comment:
I'm not keen on adding a print statement. As you highlight, this will not be shown by default anyway. The tests will be skipped, and to see details of skipped tests the user is expected to add `-rs` to the command line as documented [here](https://docs.pytest.org/en/latest/skipping.html).

I would be okay with adding a single visible warning before the tests if the base URL matches the sensitive URL.